### ISSUE TITLE
wwen:get-episodes-for-shows - Undefined index: notes

### ISSUE THREAD

**Issue by networkscraper** [networkscraper/networkscraper#2]
When the wwen:get-episodes-for-shows command is run it gets the following error when it hits 'WWe Quick Hits 2':

```
WWE Quick Hits 2



  [ErrorException]
  Undefined index: notes
```

**Comment by networkscraper** [comment]
Looks like the issue is that the this episode does not have a "notes" attribute.

**Comment by networkscraper** [comment]
Fixed in c6e042d0f215ce15bcafccb82bafbbe39755db3f
